pkg/cloud/azure/services/disks/mock_disks/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@bazel_gomock//:gomock.bzl", "gomock")
+
+gomock(
+    name = "mocks",
+    out = "disks_mock.go",
```

Comment:
can you please checkin this file as well

Reply:
added